Update github-trademark-policy.md

todlove opened the pull request:
[moderator removed comment]
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

Closes [issue link]

<!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed:

<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [x] This pull request impacts the contribution experience
  - [x] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

<!-- Description of the writer impact here -->

todlove committed:
Update github-trademark-policy.md

https://gist.github.com/GabrielMMelo/0e146f32d73978bf0d0a06786bcbc96c#gistcomment-3776249

Ti6 commented:
Hola world thanks peace

On Fri, Jun 11, 2021, 12:17 AM OPEN​TERMUX​ ***@***.***>
wrote:

>
> https://gist.github.com/GabrielMMelo/0e146f32d73978bf0d0a06786bcbc96c#gistcomment-3776249
> Why:
>
> Closes [issue link]
> What's being changed: Check off the following:
>
>    - I have reviewed my changes in staging (look for the
>    *deploy-to-heroku* link in your pull request, then click *View
>    deployment*).
>    - For content changes, I have completed the self-review checklist
>    <https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review>.
>
> Writer impact (This section is for GitHub staff members only):
>
>    - This pull request impacts the contribution experience
>       - I have added the 'writer impact' label
>       - I have added a description and/or a video demo of the changes
>       below (e.g. a "before and after video")
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/github/docs/pull/7281
> Commit Summary
>
>    - Update github-trademark-policy.md
>
> File Changes
>
>    - *M* content/github/site-policy/github-trademark-policy.md
>    <https://github.com/github/docs/pull/7281/files#diff-7fdc83c3443444002f360364462d925a8fdc1bd5e354b78b37eb8473cb119de0>
>    (11)
>
> Patch Links:
>
>    - https://github.com/github/docs/pull/7281.patch
>    - https://github.com/github/docs/pull/7281.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/7281>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUM2YKNRMNEMVR7LNVZM4QDTSGTARANCNFSM46QAJLOQ>
> .
>

todlove committed:
Merge branch 'main' into patch-2

todlove commented:
เมื่อ วันศุกร์ที่ 11 มิถุนายน ค.ศ. 2021, Ti6 ***@***.***>
เขียนว่า:

> Hola world thanks peace
>
> On Fri, Jun 11, 2021, 12:17 AM OPEN​TERMUX​ ***@***.***>
> wrote:
>
> >
> > https://gist.github.com/GabrielMMelo/0e146f32d73978bf0d0a06786bcbc9
> 6c#gistcomment-3776249
> > Why:
> >
> > Closes [issue link]
> > What's being changed: Check off the following:
> >
> > - I have reviewed my changes in staging (look for the
> > *deploy-to-heroku* link in your pull request, then click *View
> > deployment*).
> > - For content changes, I have completed the self-review checklist
> > <https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review>.
> >
> > Writer impact (This section is for GitHub staff members only):
> >
> > - This pull request impacts the contribution experience
> > - I have added the 'writer impact' label
> > - I have added a description and/or a video demo of the changes
> > below (e.g. a "before and after video")
> >
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/github/docs/pull/7281
> > Commit Summary
> >
> > - Update github-trademark-policy.md
> >
> > File Changes
> >
> > - *M* content/github/site-policy/github-trademark-policy.md
> > <https://github.com/github/docs/pull/7281/files#diff-
> 7fdc83c3443444002f360364462d925a8fdc1bd5e354b78b37eb8473cb119de0>
> > (11)
> >
> > Patch Links:
> >
> > - https://github.com/github/docs/pull/7281.patch
> > - https://github.com/github/docs/pull/7281.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/github/docs/pull/7281>, or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/
> AUM2YKNRMNEMVR7LNVZM4QDTSGTARANCNFSM46QAJLOQ>
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/7281#issuecomment-859310163>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARZ3XJAT652DCYYKUEGMAFDTSGUBFANCNFSM46QAJLOQ>
> .
>

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:

* Redistributions of source code must retain the above copyright
  notice, this list of conditions and the following disclaimer.
* Redistributions in binary form must reproduce the above copyright
  notice, this list of conditions and the following disclaimer in the
  documentation and/or other materials provided with the distribution.
* Neither the name of OVH SAS nor the
  names of its contributors may be used to endorse or promote products
  derived from this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY OVH SAS AND CONTRIBUTORS ``AS IS'' AND ANY
EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED. IN NO EVENT SHALL OVH SAS AND CONTRIBUTORS BE LIABLE FOR ANY
DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
